src/groupeddataframe/grouping.jl

```diff
@@ -99,6 +106,33 @@ julia> last(gd)
 │ 1   │ 4     │ 1     │ 4     │
 │ 2   │ 4     │ 1     │ 8     │
 
+julia> gd[(a=3,)]
+2×3 SubDataFrame
+│ Row │ a     │ b     │ c     │
+│     │ Int64 │ Int64 │ Int64 │
+├─────┼───────┼───────┼───────┤
+│ 1   │ 3     │ 2     │ 3     │
+│ 2   │ 3     │ 2     │ 7     │
+
+julia> gd[(3,)]
+2×3 SubDataFrame
+│ Row │ a     │ b     │ c     │
+│     │ Int64 │ Int64 │ Int64 │
+├─────┼───────┼───────┼───────┤
+│ 1   │ 3     │ 2     │ 3     │
+│ 2   │ 3     │ 2     │ 7     │
+
+julia> k = first(keys(gd))
+GroupKey: (a = 3)
+
+julia> gd[k]
+2×3 SubDataFrame
+│ Row │ a     │ b     │ c     │
+│     │ Int64 │ Int64 │ Int64 │
+├─────┼───────┼───────┼───────┤
+│ 1   │ 3     │ 2     │ 3     │
+│ 2   │ 3     │ 2     │ 7     │
+
```

Comment:
maybe add one example with array indexing (returning a `GroupedDataFrame`)?

Reply:
I'll probably do this in the follow-up, which expands array indexing.

Reply:
OK